### PR TITLE
Added kingfisher pod

### LINKED ISSUE #39: [iOS] Cache images called from the API
Add Kingfisher to call the images from the API
[Kingfisher](https://github.com/onevcat/Kingfisher)

### Acceptance Criteria
- [x] Install the kingfisher library
- [x] Replace image implementation with the kingfisher library
- [x] Remove old implementation to call from an URL
- [x] Ensure that is caching the images and don´t downloading again (check the loading speed)

### PR DESCRIPTION
**Pull request by IrvingJVR-MX**
Added kingfisher pod, also the getPokemonsImageBy method was modified so now it returns a URL

### CODE REVIEW

**Review by Rodrigo-Schar — APPROVED** [review]

File: Podfile [hunk 1]
```diff
@@ -7,7 +7,8 @@ target 'Pokedex_ios' do
 
   # Pods for Pokedex_ios
   pod 'SwiftLint', '0.47.1'
-
+  pod 'Kingfisher'
```
Comment (jmacboy) [inline Podfile]: don't forget to always add the version on the pods

**Review by jmacboy — CHANGES_REQUESTED** [review]
just that and we're good

File: Podfile (hunk 1 above)
Comment (IrvingJVR-MX) [reply Podfile]: done :)

**Comment by jmacboy** [comment]
merge the latest from develop on this branch @IrvingJVR-MX  and youre good to go

**Review by jmacboy — APPROVED** [review]

**Pull request merged**